onnxruntime/core/providers/qnn/builder/qnn_model.cc

```diff
@@ -185,15 +185,41 @@ Status QnnModel::SetupQnnInputOutput(const logging::Logger& logger) {
   return Status::OK();
 }
 
-Status QnnModel::ExecuteGraph(const Ort::KernelContext& context, const logging::Logger& logger) {
+static Status BindQnnTensorMemoryToOrtValue(const logging::Logger& logger,
+                                            QnnBackendManager& qnn_backend_manager,
+                                            const OrtMemoryInfo& ort_value_memory_info,
+                                            void* ort_value_data, uint32_t ort_value_data_size,
+                                            Qnn_ContextHandle_t qnn_context,
+                                            Qnn_Tensor_t& qnn_tensor) {
```

Comment:
nit: would have expected something called Bind...ToOrtValue to take the OrtValue. Could it, or is it more 'bind to ORT allocated memory'?

Reply:
this function should be called with OrtValues, or data from OrtValues. it's a little tricky because at the call sites we actually have Ort::ConstValue and Ort::UnownedValue. I'll rename it.